x/oracle/keeper/msg_server_create_price.go

```diff
@@ -0,0 +1,52 @@
+package keeper
+
+import (
+	"context"
+	"strconv"
+
+	"github.com/ExocoreNetwork/exocore/x/oracle/types"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+// CreatePrice proposes price for new round of specific tokenFeeder
+func (ms msgServer) CreatePrice(goCtx context.Context, msg *types.MsgCreatePrice) (*types.MsgCreatePriceResponse, error) {
```

Comment:
Wouldn't this be called by validator?

Reply:
This serves a a message service, and the message should be signed by validator